### DISCUSSION TITLE
Create variables with helpers like {{$guid}}

### DISCUSSION THREAD

**Discussion by eddswitchengage** [postmanlabs/postman-flows#173] [Ideas]
It would be extremely useful for automatically creating test data if we were able to access the variable expressions that are available in collections such as {{$guid}} and {{$timestamp}}.

**Comment by akshay2000** [comment]
One workaround I've found is to send request containing `{{$guid}}` etc. to the Postman echo endpoint. Then create durable variables from the response.

**Comment by saswatds** [comment]
@eddswitchengage Thanks for the feedback. I am adding this as a feature request for the near term releases.
